config/build.ts

```diff
@@ -40,31 +39,50 @@ const defaultApiSets = {
 };
 
 
-(() => {
-    Promise.resolve()
-        .then(processSnippets)
+(async() => {
```

Comment:
Ricky, I am not 100% sure that this code is fully ready yet, I know that Shashwat is working on it.  You may be best off waiting till later today, or merging in (cherry-picking?) only the samples files but not the build tool.  Or at least, making sure that the playlist produced by the new changes to the tooling is no different than it was before.